components/ProfileMain/ProfileMain.jsx

```diff
@@ -1,73 +1,115 @@
+              collapsed={isCollapsed}
+              onCollapse={() => onCollapse(isCollapsed)}
+            >
+              <NewCourseNavbar isCollapsed={isCollapsed} />
+            </Sider>
           </div>
-          <div className="col-sm-9 col-md-10">
-            <div className="col-12 сol-sm-6 profile-welcome">
-              <div className="profile-welcome-block">
-                <h3 className="profile-welcome-title">{t('title')}, {user.name}</h3>
-                <p className="profile-welcome-subtitle">{t('subtitle')}</p>
-              </div>
-              <div className="col-12 col-sm-6">
-                <img className="profile-welcome-img" src="/7.png" alt="upgrade"/>
-              </div>
+          <Content>
+            <div className='d-block d-md-none'>
+              <NewCourseNavbar isCollapsed={isCollapsed} />
             </div>
-            <div className="profile-courses">
-              <h3 className="profile-courses-title">{t('myCourses')}</h3>
-              {courseLearningBlock.length !== 0
-                ? courseLearningBlock
-                : (
-                  <div className="profile-courses-one d-flex my-3">
-                    <img className="profile-courses-nocourses-img" src="/9.png" alt="preview-course-photo"/>
-                    <div className="profile-courses-one-content">
-                      <h3 className="profile-courses-one-title" style={{ fontSize: '18px' }}>{t('ohUh')}</h3>
-                      <p className="profile-courses-one-text">{t('noCourse')}</p>
+            <div className='container'>
+              <div className='col-12 сol-sm-6 profile-welcome'>
+                <div className='profile-welcome-block'>
+                  <h3 className='profile-welcome-title'>
+                    {t('title')}, {user.name}
+                  </h3>
+                  <p className='profile-welcome-subtitle'>{t('subtitle')}</p>
+                </div>
+                <div className='col-12 col-sm-6'>
+                  <img
+                    className='profile-welcome-img'
+                    src='/7.png'
+                    alt='upgrade'
+                  />
+                </div>
+              </div>
+              <div className='profile-courses'>
+                <h3 className='profile-courses-title'>{t('myCourses')}</h3>
+                {courseLearningBlock.length !== 0 ? (
+                  courseLearningBlock
+                ) : (
+                  <div className='profile-courses-one d-flex my-3'>
+                    <img
+                      className='profile-courses-nocourses-img'
+                      src='/9.png'
+                      alt='preview-course-photo'
+                    />
+                    <div className='profile-courses-one-content'>
+                      <h3
+                        className='profile-courses-one-title'
+                        style={{ fontSize: '18px' }}
+                      >
+                        {t('ohUh')}
```

Comment:
`ohUh`? When I reading it, so many Gachi movies in my mind....

 Please, make it more understandable